packages/components/src/common/Notifications/NotificationDismissButton/NotificationDismissButton.module.css

```diff
@@ -0,0 +1,11 @@
+.dismiss {
+  @apply w-12 h-12 justify-center rounded-full;
+}
+
+.dismiss:hover {
+  /* The hover transition is inherited from the Button component */
+  @apply text-white;
+
+  /* --dismiss-hover-color variable should be set in the consumer's styles */
+  background-color: var(--dismiss-hover-color);
+}
```

Comment:
ooh this abstraction is nice. I'm wondering if this component is actually general enough to be something like `common/CloseButton`? Thinking ahead to a Modal close button, I imagine we'll use basically this same design?

Reply:
You got it 👍